packages/@react-aria/landmark/src/useLandmark.ts

```diff
@@ -242,10 +245,10 @@ class LandmarkManager {
     if (currentLandmark && currentLandmark.ref.current !== e.target) {
       this.updateLandmark({...currentLandmark, lastFocused: e.target as HTMLElement});
```

Comment:
I think you had a good change here
```suggestion
      this.updateLandmark({ref: currentLandmark.ref, lastFocused: e.target as HTMLElement});
```